presto-verifier/src/main/java/com/facebook/presto/verifier/framework/ExplainVerification.java

```diff
@@ -45,7 +50,7 @@
         extends AbstractVerification<QueryBundle, ExplainMatchResult, String>
 {
     private static final ResultSetConverter<String> QUERY_PLAN_RESULT_SET_CONVERTER = resultSet -> Optional.of(resultSet.getString("Query Plan"));
-    private static final JsonCodec<JsonRenderedNode> PLAN_CODEC = jsonCodec(JsonRenderedNode.class);
+    private static JsonCodec<JsonRenderedNode> PLAN_CODEC;
```

Comment:
renaming it to planCodec should be done here in this commit, not the last one.

Reply:
fixed